addons/event/models/event_mail.py

```diff
@@ -151,13 +120,9 @@ def _compute_mail_state(self):
             else:
                 scheduler.mail_state = 'running'
 
-    @api.constrains('notification_type', 'template_ref')
-    def _check_template_ref_model(self):
-        model_map = self._selection_template_model_get_mapping()
-        for record in self.filtered('template_ref'):
-            model = model_map[record.notification_type]
-            if record.template_ref._name != model:
-                raise ValidationError(_('The template which is referenced should be coming from %(model_name)s model.', model_name=model))
+    @api.depends('template_ref')
+    def _compute_notification_type(self):
```

Comment:
Should we align the docstring?

Reply:
Here you just meant use the same docstring for event type and event mail?
Did that